### PR TITLE
feat: added rating, toast & grid component

### PR DESCRIPTION
**Pull request by keshavkumar-in**
Added Rating, Toast & Grid Components

### CODE REVIEW

File: lib/package/components/grid.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@charset "UTF-8";
+
+.grid {
+  display: grid;
+  gap: 1rem;
+}
+
+.grid-col-two {
+  grid-template-columns: repeat(2, minmax(0, 1fr));
```
Comment (sainath7878) [inline lib/package/components/grid.css:9]: you can just write 1fr here. minmax is not required I guess.

**Review by sainath7878 — COMMENTED** [review]
Rest looks good.

**Review by 05him — APPROVED** [review]
your code looks clean to me .. 
couldn't find anything to suggest
so its Good To Go

edit:- I forget to mention that this code snippet looks a bit messy , coz the line number and code are getting mixed up
image

**Comment by keshavkumar-in** [comment]
> your code looks clean to me .. couldn't find anything to suggest so its Good To Go
> 
> edit:- I forget to mention that this code snippet looks a bit messy , coz the line number and code are getting mixed up image

Yeah, it's a carbon code snippets. Sometimes had to refresh to get it the correct way. Thanks for noting.

**Pull request merged**